src/cascadia/TerminalApp/App.cpp

```diff
@@ -1307,8 +1315,15 @@ namespace winrt::TerminalApp::implementation
                                             _settings->GlobalSettings().GetDefaultProfile();
         const auto controlSettings = _settings->MakeSettings(realGuid);
 
-        // Create a Conhost connection based on the values in our settings object.
-        TerminalConnection::ITerminalConnection controlConnection = TerminalConnection::ConhostConnection(controlSettings.Commandline(), controlSettings.StartingDirectory(), 30, 80, winrt::guid());
+        TerminalConnection::ITerminalConnection controlConnection{ nullptr };
+        if (controlSettings.Commandline() == L"Azure")

```

Comment:
Using a string as a magic value in an existing field is probably not a great idea.

We should probably have another parameter that is some sort of enum or guid that specifies which connection type this is and pipe it all the way through settings.

Reply:
I put it here because the Azure connection does not actually need a Commandline parameter (but Conhost connection does and so all the settings schemes contain it). So if we add another parameter, the settings.Commandline variable for non-conhost connections will probably just have to be 'n/a' or something, so I figured it might make sense to just use that as an indicator instead. Should I just change it to check for the guid in the settings?

Reply:
So, here's the thought. We're not yet ready to support multiple "connection types," and I don't want us to start something and then start on the wrong foot; this is a stopgap, yes, but perhaps it's good enough to stop us from misdesigning the early extensibility model.